terminal-sample/src/extension.ts

```diff
@@ -180,6 +180,15 @@ export function activate(context: vscode.ExtensionContext) {
 			}
 		});
 	}));
+
+	context.subscriptions.push(vscode.window.registerTerminalProfileProvider('terminalTest.terminal-profile', {
+		provideTerminalProfile(token: vscode.CancellationToken): vscode.ProviderResult<vscode.TerminalOptions> {
```

Comment:
Awaiting type checks introduced in the [1.58 release of microsoft/vscode](https://github.com/microsoft/vscode-extension-samples/milestone/23).